universal-login-wallet/src/app/constants/urlMapping.ts

```diff
@@ -1,3 +1,11 @@
+const routesForDeployed = [
```

Comment:
I would name it differently e.g. `urlsForDeployed`, `possibleUrlForDeployed`, `pathsForDeployed` or something different